core/trino-main/src/main/java/io/trino/sql/planner/SortExpressionExtractor.java

```diff
@@ -88,7 +88,7 @@ private static SortExpressionContext merge(SortExpressionContext left, SortExpre
     }
 
     private static class SortExpressionVisitor
-            extends IrVisitor<Optional<SortExpressionContext>, Void>
+            extends IrVisitor<Optional<List<SortExpressionContext>>, Void>
```

Comment:
`Optional` is no longer needed, just `List<SortExpressionContext>`

Reply:
updated